agimus_demo_05_pick_and_place/launch/bringup.launch.py

```diff
@@ -36,6 +42,37 @@ def launch_setup(
         executable="agimus_controller_node",
         parameters=[get_use_sim_time(), agimus_controller_yaml],
         output="screen",
+        remappings=[("robot_description", "robot_description_with_collision")],
+    )
+
+    environment_description = ParameterValue(
+        Command(
+            [
+                PathJoinSubstitution([FindExecutable(name="xacro")]),
+                " ",
+                PathJoinSubstitution(
+                    [
+                        FindPackageShare("agimus_demo_05_pick_and_place"),
+                        "urdf",
+                        "obstacles.xacro",
```

Comment:
minor comment: environment.xacro?
```suggestion
                        "environment.xacro",
```